udc/src/main/java/io/crate/udc/ping/PingTask.java

```diff
@@ -102,6 +102,11 @@ String isEnterprise() {
         return SharedSettings.ENTERPRISE_LICENSE_SETTING.setting().getRaw(settings);
     }
 
+    @VisibleForTesting
+    String getLicenseIdent() {
+        return SharedSettings.LICENSE_IDENT_SETTING.setting().get(settings);
```

Comment:
this won't give you the actual maybe dynamically changed setting, it would only give you the setting defined on node start. to get the actual runtime setting, use a instance var, add a setter method and register an update consumer to the clusterSettings using this setter.